csharp/ql/src/semmle/code/csharp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -255,6 +255,17 @@ private module Cached {
    */
   cached
   predicate localFlowStepImpl(Node nodeFrom, Node nodeTo) {
+    simpleLocalFlowStep(nodeFrom, nodeTo)
+  }
+
+  /**
+   * INTERNAL: do not use.
+   *
+   * This is the local flow predicate that's used as a building block in global
+   * data flow. It may have less flow than the `localFlowStep` predicate.
+   */
+  cached
+  predicate simpleLocalFlowStep(Node nodeFrom, Node nodeTo) {
```

Comment:
I would rather if you rename the original `localFlowStepImpl` to `simpleLocalFlowStep`, and then change the alias in `DataFlowPublic.qll` to `predicate localFlowStep = simpleLocalFlowStep/2;`

Reply:
Done (except it's not an alias -- see commit message).